dynamic_programming/house_robber.cpp

```diff
@@ -0,0 +1,95 @@
+/*House Robber - Dynamic Programming
```

Comment:
```suggestion
/**
 * @file 
 * @brief House Robber - Dynamic Programming
```
Update the documetnation block per the guidelines. A sample code is provided for your convenience. In the above comment, an example existing code was also provided as a reference.